projects/plugins/jetpack/_inc/lib/icalendar-reader.php

```diff
@@ -662,7 +662,7 @@ public function parse( $url = '' ) {
 				}
 			}
 			foreach ( $recurrences as $recurrence ) {
-				$count_vevent = count( $this->cal['VEVENT'] );
+				$count_vevent = is_countable( $this->cal['VEVENT'] ) ? count( $this->cal['VEVENT'] ) : 0;
```

Comment:
Again, we just iterated over the VEVENT arrays a few lines above, so it's a little late to be testing `is_countable` now.

Reply:
marking as false positive